src/main/java/io/fabric8/jenkins/openshiftsync/BuildSyncRunListener.java

```diff
@@ -162,7 +164,7 @@ protected void doRun() throws Exception {
           pollLoop();
         }
       };
-      Timer.get().scheduleAtFixedRate(task, pollPeriodMs, pollPeriodMs, TimeUnit.MILLISECONDS);
+      Timer.get().scheduleAtFixedRate(task, pollPeriodDelayMs, pollPeriodMs, TimeUnit.MILLISECONDS);
```

Comment:
How is this related to the issue?

Reply:
It's indirectly related. Recently similar [issue](https://github.com/openshiftio/openshift.io/issues/1648) has been observed in plugin behavior and the root cause of this issue was the frequency at which it polls the OpenShift API's.
IMHO, it would be better if this fix can go along with this issue. It may sound premature optimization though and not directly related.

Reply:
In this case, I'd recommend putting this into a dedicated commit with a proper commit message. Nothing stops you to address two issues within the same pull request, but for traceability, it makes sense to separate this. 

IMO you should create an issue for this in this project referencing https://github.com/openshiftio/openshift.io/issues/1648. Then you split this commit into to dedicated commits, each referencing its matching issue.

Reply:
> IMHO, it would be better if this fix can go along with this issue.

Why?

I mean, I understand why we want this change in, but why does it have to go "along with this issue"? 

In any case, please split the commits as Hardy suggested.